Tests/PubNubContractTest/Steps/Push/PubNubPushContractTestSteps.swift

```diff
@@ -164,6 +164,6 @@ public class PubNubPushContractTestSteps: PubNubContractTestCase {
   }
 
   fileprivate func pushServiceFromWhen(match: String) -> PubNub.PushService {
-    ["GCM", "FMC"].contains(match) ? .gcm : .apns
+    ["GCM", "FMC"].contains(match) ? .fcm : .apns
```

Comment:
should this be "FMC" ? or "FCM" ?

Reply:
Indeed, there's a typo in it. I checked the feature file for these tests and it's safe to fix it

Reply:
Done